lib/util/stream_utils.js

```diff
@@ -347,19 +347,32 @@ shaka.util.StreamUtils = class {
     await shaka.util.StreamUtils.getDecodingInfosForVariants(
         manifest.variants, usePersistentLicenses, /* srcEquals= */ false,
         /** preferredKeySystems= */ []);
+
+    const ContentType = shaka.util.ManifestParserUtils.ContentType;
+    const Capabilities = shaka.media.Capabilities;
+    const ManifestParserUtils = shaka.util.ManifestParserUtils;
+    const MimeUtils = shaka.util.MimeUtils;
+    const StreamUtils = shaka.util.StreamUtils;
+    const isXboxOne = shaka.util.Platform.isXboxOne();
+
     manifest.variants = manifest.variants.filter((variant) => {
       // See: https://github.com/shaka-project/shaka-player/issues/3860
       const video = variant.video;
 
-      const ContentType = shaka.util.ManifestParserUtils.ContentType;
-      const Capabilities = shaka.media.Capabilities;
-      const ManifestParserUtils = shaka.util.ManifestParserUtils;
-      const MimeUtils = shaka.util.MimeUtils;
-      const StreamUtils = shaka.util.StreamUtils;
+      // See: https://github.com/shaka-project/shaka-player/issues/3380
+      // Note: it makes sense to drop early
+      if (isXboxOne && video &&
+        ((video.width && video.width > 1920) ||
```

Comment:
If width is null or undefined, it will not be > 1920.  Please drop the "truthy" checks on width and height and combine width > 1920 || height > 1080 on one line if possible, to make this long sequence of booleans more readable.

Also, these continuations should be indented +2 from where they are now.  That will also make them easier to differentiate from the log inside the if().  Thanks!